src/tooltip/Tooltip.js

```diff
@@ -213,27 +217,35 @@ class Tooltip extends React.PureComponent {
   };
 
   renderStaticModalContent = () => {
-    const { withOverlay, overlayColor } = this.props;
+    const { withOverlay, overlayColor, skipAndroidStatusBar } = this.props;
 
     return (
-      <Fragment>
+      <View>
```

Comment:
Can you please explain the reason for removing the `Fragment`

Reply:
@pranshuchittora  Thought of using native view instead of Fragment .  
If you feel  fragment has better advantage then view, let me know i will change it back to fragment.

Reply:
In this scenario, you are not using `View` for any layout, it's been used as a container. The advantage of using `Fragment` is that it doesn't populate the actual UI. It gets omitted by the React reconcile while evaluating the tree. 
For more info -> https://reactjs.org/docs/fragments.html

Reply:
@pranshuchittora  Thank you very much for the suggestion . I have changed it back to Fragment :+1: